### DISCUSSION TITLE
Git clone Error

### DISCUSSION THREAD

**Discussion by CyntTech** [community/community#117600] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

I need help on how to resolve this git clone error: Cloning into 'SchoolBaseProject'...
remote: Enumerating objects: 290, done.
remote: Counting objects: 100% (290/290), done.
remote: Compressing objects: 100% (210/210), done.
error: RPC failed; curl 92 HTTP/2 stream 0 was not closed cleanly: CANCEL (err 8)
error: 6692 bytes of body are still expected
fetch-pack: unexpected disconnect while reading sideband packet
fatal: early EOF
fatal: fetch-pack: invalid index-pack output

**Comment by MuhammadAwais09** [comment]
git config --global http.postBuffer 52428800
sudo git clone https://github.com/pdfcpu/pdfcpu --depth 1
